docs/graphql/manual/mutations/insert.rst

```diff
@@ -213,26 +213,32 @@ Insert an object and get a nested object in response
       }
     }
 
+Insert related data through relationships
+-----------------------------------------
+**Example:** Insert an ``author``, their ``address`` and an ``article``.
 
-Insert an object and its nested object in the same mutation
------------------------------------------------------------
-**Example:** Insert a new ``article`` object with its ``author`` and return the inserted article object with its author
-in the response
+Let's say an ``author`` has an ``object relationship`` called ``address`` to the ``addresses`` table and an ``array relationship`` called ``articles`` to the ``articles`` table.
 
 .. graphiql::
   :view_only:
   :query:
-    mutation insert_article {
-      insert_article(
-        objects: [
+    mutation insertData {
+      insert_authors
+        (objects: [
           {
-            id: 21,
-            title: "Article 1",
-            content: "Sample article content",
-            author: {
+            id: 11,
+            name: "John",
+            address: {
+              data: {
+                id: 12,
+                location: "San Francisco"
+              }
+            },
+            articles: {
```

Comment:
can we add 2 articles here instead of just 1. I remember some user mentioning that they thought you could just add one nested object at a time based on just seeing the example. lets not ever have to answer that again